django/conf/__init__.py

```diff
@@ -30,6 +31,35 @@
 )
 
 
+def add_script_prefix(value):
+    """
+    Prefix header value SCRIPT_NAME to ``value`` under certain circumstances.
+
+    Useful for when the app is being served at a subpath and manually prefixing
+    that subpath to STATIC_URL and STATIC_ROOT in settings is inconvenient
+    (i.e., a deployment concern). Refs #25598.
+    """
+    try:
+        URLValidator()(value)
+        return value
+    except (ValidationError, AttributeError):
+        pass
+    try:
+        value = value.strip()
```

Comment:
Under which circumstances is this useful? I'd expect that value to be correct all the time and if it start with a blank so be it.

Reply:
Removed, see #11564